### ISSUE TITLE
Kisscartoon is now kimcartoon

### ISSUE THREAD

**Issue by ThatLurker** [wileyyugioh/kissanime_dl#31]
kimcartoon.me is the new address

**Comment by milezzz** [comment]
Any chance @wileyyugioh will save us guys??

DAMN this was such a phenomenal little tool! I've tried it with kisscartoon.io AND kimcartoon.me and neither work.

**Comment by ThatLurker** [comment]
He's currently waiting on selenium with headless chrome to be released.

**Comment by wileyyugioh** [comment]
I think I'm going to put development on hold for a while. Kissanime is making it very troublesome to download videos, and with all the site developments such as the sketchy ads and video hosts using even more obfuscation, I'm not sure how stable the program will be even if it were to use a headless browser, as well as the safety of using an automated script.
